src/main/java/de/tum/in/www1/artemis/domain/ProgrammingExerciseBuildConfig.java

```diff
@@ -0,0 +1,200 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.util.Objects;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import de.tum.in.www1.artemis.service.connectors.aeolus.Windfile;
+import de.tum.in.www1.artemis.service.connectors.vcs.AbstractVersionControlService;
+
+@Entity
+@Table(name = "programming_exercise_build_config")
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class ProgrammingExerciseBuildConfig extends DomainObject {
+
+    private static final Logger log = LoggerFactory.getLogger(ProgrammingExerciseBuildConfig.class);
+
+    @Column(name = "sequential_test_runs")
+    private Boolean sequentialTestRuns;
```

Comment:
```suggestion
    private boolean sequentialTestRuns;
```
This way we can also get rid of the json annotation for the getter

Reply:
This has to be `Boolean`, since the field was of that type. When performing the migration, older null values will be copied to the new table, which will cause an error